typescript-light-bulb/styles.css

```diff
@@ -0,0 +1,32 @@
+.background {
+  background-color: black;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+}
+
+.background-off {
+  background-color: black;
+}
+
+.background-on {
+  background-color: rgb(255, 250, 230);
+}
+
+.bulb {
+  width: 250px;
+  height: 250px;
+  border-radius: 125px;
+}
+
+.bulb-off {
+  background-color: rgb(25, 25, 25);
+  box-shadow: none;
+}
+
```

Comment:
You could still give this a shadow for practice styling it. If you look closely at the example gif, there's actually a solid, dark shadow above the lightbulb.